### PR TITLE
CCX : Add user sync url

### PR DESCRIPTION
**Pull request by mjaworskiccx**

### CODE REVIEW

File: static/bidder-info/ccx.yaml [hunk 1]
```diff
@@ -8,7 +8,6 @@ capabilities:
       - banner
       - video
 userSync:
-  # Clickonometrics supports user syncing, but requires configuration by the host. contact this
-  # bidder directly at the email address in this file to ask about enabling user sync.
-  supports:
-    - redirect
+  redirect:
+    url: "https://sync.clickonometrics.pl/prebid/set-cookie?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirurl={{.RedirectURL}}"
+    userMacro: "${USER_ID}"
```
Comment (gargcreation1992) [inline static/bidder-info/ccx.yaml:13]: The redirectURL doesn't seem to be setting the `${USER_ID}` macro. Could you please check? The below is the URL that I am trying to reach to - 
```
https://sync.clickonometrics.pl/prebid/set-cookie?gdpr=&gdpr_consent=&us_privacy=&redirurl=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dccx%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24%7BUSER_ID%7D
```

**Review by gargcreation1992 — COMMENTED** [review]
Please check the userMacro and redirect url.

**Comment by mjaworskiccx** [comment]
Please check now

**Review by Sonali-More-Xandr — APPROVED** [review]
Redirect url is working for me now, also setting the UID.

File: static/bidder-info/ccx.yaml (hunk 1 above)
Comment (gargcreation1992) [reply static/bidder-info/ccx.yaml:13]: It is working now!

**Review by gargcreation1992 — APPROVED** [review]

**Pull request merged**